resources/netregexes.ts

```diff
@@ -31,37 +27,70 @@ const keysThatRequireTranslation = [
   'line',
 ];
 
+type ParseHelperField<
+  T extends TriggerTypes,
+  T2 extends NetFieldsReverse[T],
+  field extends keyof T2
+> = {
+  field: T2[field] extends string ? T2[field] : never;
+  value?: string;
+};
+
+type ParseHelperFields<T extends TriggerTypes> = {
+  [field in keyof NetFieldsReverse[T]]: ParseHelperField<T, NetFieldsReverse[T], field>;
+};
+
+const test: ParseHelperFields<'StartsUsing'> = {
+  0: { field: 'type', value: '20' },
+  1: { field: 'timestamp' },
+  2: { field: 'sourceId' },
+  3: { field: 'source' },
+  4: { field: 'id' },
+  5: { field: 'ability' },
+  6: { field: 'targetId' },
+  7: { field: 'target' },
+  8: { field: 'castTime' },
+  9: { field: 'x' },
+  10: { field: 'y' },
+  11: { field: 'z' },
+  12: { field: 'heading' },
+};
+test;
```

Comment:
Was this leftover from debugging?

Reply:
Yeah, this was leftover from me trying to figure out some weirdness with the `ParseHelperField` type.